### PR TITLE
change default registry from docker to cplane

### PR DESCRIPTION
**Pull request by kbknapp**
This PR changes the default registry from `registry.hub.docker.com/` to `registry.cplane.cloud/`

### CODE REVIEW

**Comment by kbknapp** [comment]
Current dependencies on/for this PR:
* main
  * **PR #254** <a href="https://app.graphite.dev/github/pr/seaplane-io/seaplane/254" target="_blank">Graphite</a> 
    * **PR #255** <a href="https://app.graphite.dev/github/pr/seaplane-io/seaplane/255" target="_blank">Graphite</a>  👈
      * **PR #256** <a href="https://app.graphite.dev/github/pr/seaplane-io/seaplane/256" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/seaplane-io/seaplane/255?utm_source=stack-comment).

File: seaplane-cli/tests/api/formation.rs [hunk 1]
```diff
@@ -45,15 +45,15 @@ fn default_cfg_json() -> serde_json::Value {
         "connections":[],
         "flights":[{
             "name":"flask",
-            "image":"registry.hub.docker.com/stubb/alpine:latest",
+            "image":"registry.cplane.cloud/stubb/alpine:latest",
```
Comment (alexthecarp) [inline seaplane-cli/tests/api/formation.rs:48]: @kbknapp these won't work like this, since the registry expects "tenant ID" (as in `tnt-xxxxxxxx...`).

Comment (kbknapp) [reply seaplane-cli/tests/api/formation.rs:48]: That's fine - these tests are hitting a mock right that's really just asserting the base URL is set correctly. `stubb` could be a stand-in for tenant ID until I have the time to circle back.

File: docs/CONFIGURATION_SPEC.md [hunk 2]
```diff
@@ -90,10 +90,10 @@ may contain one of the following values:
 Names which container image registry will be inferred if not provided for
 commands and CLI flags that take a container image reference.
 
-Default value: `registry.hub.docker.com/`
+Default value: `registry.cplane.cloud/`
```
Comment (ericsampson) [inline docs/CONFIGURATION_SPEC.md:93]:
```suggestion
Default value: `registry.cplane.cloud`
```

**Pull request merged**